digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/RoadNameServiceSpec.scala

```diff
@@ -116,7 +116,7 @@ class RoadNameServiceSpec extends FunSuite with Matchers {
 
   test("Fetch updated road names, no updates") {
     runWithRollback {
-      val result = roadNameService.getUpdatedRoadNamesInTX(DateTime.parse("3001-01-01"))
+      val result = roadNameService.getUpdatedRoadNames(DateTime.parse("3001-01-01"), None)
```

Comment:
Why this is starting a new transaction? I think getUpdatedRoadNamesInTX should be used instead.

Reply:
Yeah its unnecessary, I will remove it